webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require('path');
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+let config = {
+	entry: './src/client/index.js',
+	output: {
+		path: path.join(__dirname, '/build'),
+		filename: 'index_bundle.js',
+		publicPath: '/'
+	},
+	module: {
+		loaders:[
+			{ test: /\.(js)$/, use: 'babel-loader' }
```

Comment:
when npm calls "npm run build" it will run webpack and webpack will run whatever modules/loaders are listed here. babel-loader just runs babel with whatever transformations you specified in your presets (set presets either in .babelrc or package.json)